File: Expressions/financial_calculator.py
# Anthony Petersen, Financial Calculator

#welcome user and explains programs use
print("Welcome, this is a financial calculator to calculate your expenses, savings, and spendings")
#have user input monthly income
income = float(input("What is your income in number amount: "))
#have user input cost of rent
rent = float(input("How much does your rent cost: "))
#have user input cost of utilities
utilities = float(input("How much does your utilities such as power, water, etc. cost: "))
#have user input cost of groceries
groceries = float(input("How much do your groceries cost on average monthly: "))
#have user input cost of transportation
transportation = float(input("How much does transportation cost monthly: "))
#calculate how much they put into savings (10% of income)(income*.1)
savings = income*.1
#calculate how much is spending(income minus savings and expenses)
spending_money = float(income - savings - rent - utilities - groceries - transportation)
#calculate percent of income used for rent (rent/income *100)
percent_rent = float(rent/income *100)
#calculate percent of income used for utilities (utilities/income *100)
percent_utilities = float(utilities/income *100)
#calculate percent of income used for groceries (groceries/income *100)
percent_groceries = float(groceries/income *100)
#calculate percent of income used for transportation (transportation/income *100)
percent_transportation = float(transportation/income *100)
#calculate percent of income used for savings (savings/income *100)
percent_savings = float(savings/income *100)
#calculate percent of income used for spending (spending/income *100)
percent_spending = float(spending_money/income *100)
#print the following

    #your rent is $ which is % of your income
print("Your rent is", rent, "which is",percent_rent,"% of your income")
    #your utilities is $ which is % of your income
print("Your utilties is", utilities, "which is",percent_utilities,"% of your income")
    #your groceries is $ which is % of your income
print("Your groceries is", groceries, "which is",percent_groceries,"% of your income")
    #your transportation is $ which is % of your income
print("Your transportation is", transportation, "which is",percent_transportation,"% of your income")
    #your savings is $ which is % of your income
print("Your savings is", savings, "which is",percent_savings,"% of your income")
    #your spending is $ which is % of your income
print("Your spending money leftover is", spending_money, "which is",percent_spending,"% of your income")

File: Expressions/financial_calculator.c
// Anthony Petersen, Financial Calculator
#include <stdio.h>
#include <math.h>

float income;
float rent;
float utilities;
float groceries;
float transportation;

int main(void){
//welcome user and explains programs use
    printf("Welcome to your financial calculator, which will calculate your expenses, savings, and spending money \n");
//have user input monthly income
    printf("How much is your monthly income: ");
    scanf("%f", &income);
//have user input cost of rent
    printf("How much is your monthly rent cost: ");
    scanf("%f", &rent);
//have user input cost of utilities
    printf("How much is your monthly utilities cost: ");
    scanf("%f", &utilities);
//have user input cost of groceries
    printf("How much is your monthly groceries cost: ");
    scanf("%f", &groceries);
//have user input cost of transportation
    printf("How much is your monthly transportation cost: ");
    scanf("%f", &transportation);
    //calculate percent of income used for savings
    float savings = (income)*.1;
//calculate percent of income used for rent (rent/income *100)
    float percent_rent = (rent / income) * 100;
//calculate percent of income used for utilities (utilities/income *100)
    float percent_utilities = (utilities / income) * 100;
//calculate percent of income used for groceries (groceries/income *100)
    float percent_groceries = (groceries / income) * 100;
//calculate percent of income used for transportation (transportation/income *100)
    float percent_transportation = (transportation / income) * 100;
//calculate percent of income used for savings (savings/income *100)
    float percent_savings = (savings / income) * 100;
//calculate how much is spending(income minus savings and expenses)
    float spending = income-(rent+utilities+groceries+transportation+savings);
//calculate percent of income used for spending
    float percent_spending = (spending/income)* 100;
//printf the following

    //your rent is $ which is % of your income
    printf("Your rent costs $%.2f, which is %.0f percent of your income \n", rent, percent_rent);
    //your utilities is $ which is % of your income
    printf("Your utilites cost $%.2f, which is %.0f percent of your income \n", utilities, percent_utilities);
    //your groceries is $ which is % of your income
    printf("Your groceries cost $%.2f, which is %.0f percent of your income \n", groceries, percent_groceries);
    //your transportation is $ which is % of your income
    printf("Your transportation costs $%.2f, which is %.0f percent of your income \n", transportation, percent_transportation);
    //your savings is $ which is % of your income
    printf("You're saving $%.2f, which is %.0f percent of your income \n", savings, percent_savings);
    //your spending is $ which is % of your income
    printf("Your spending money is $%.2f, which is %.0f percent of your income \n", spending, percent_spending);
    return 0;
}